src/util/rpc/tcp_client.hpp

```diff
@@ -55,7 +55,7 @@ namespace cbdc::rpc {
         /// starts the response handler thread.
         /// \return true.
         [[nodiscard]] auto init() -> bool {
-            if(!m_net.cluster_connect(m_server_endpoints, false)) {
```

Comment:
I saw this line a few weeks ago and i was banging my head trying to figure out what i was missing!